lib/components.dart

```diff
@@ -14,12 +14,16 @@ export 'src/components/text_component.dart';
 export 'src/timer.dart';
 export 'joystick.dart';
 
+export 'src/components/mixins/collidable.dart';
 export 'src/components/mixins/draggable.dart';
+export 'src/components/mixins/has_collidables.dart';
 export 'src/components/mixins/has_game_ref.dart';
 export 'src/components/mixins/hitbox.dart';
 export 'src/components/mixins/single_child_particle.dart';
 export 'src/components/mixins/tapable.dart';
 
+export 'geometry.dart';
```

Comment:
I wonder if we should export geometry by default? or make it separate like effects cc @renancaraujo

Reply:
Yeah, I think it should be separate.
I don't know why it ended up in here.